tools/alpine/push-manifest.sh

```diff
@@ -4,14 +4,11 @@ set -e
 # This script creates a multiarch manifest for the 'linuxkit/alpine'
 # image, pushes and signs it. The manifest is pushed with the tag of
 # the amd64 images (which is the suffix removed). On macOS we use the
-# credentials helper to extract the Hub credentials. We need to
-# manually sign the manifest using 'notary'.
+# credentials helper to extract the Hub credentials.
 #
 # This script is specific to 'linuxkit/alpine'. For normal packages we
 # use a different scheme.
 #
-# For signing, DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE must be set.
-#
 # This should all be replaced with 'docker manifest' once it lands.
```

Comment:
docker manifest did land a long time ago, can we use instead of manifest tool?

Reply:
We should consider it as a replacement for the embedded library stuff (although I usually prefer embedded), but that definitely should be subject of a separate PR